bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -175,6 +180,32 @@ def _setattr(self, name, val):
 
         return setattr(self, name, val)
 
+    def _run_rootlogon(self):
+        hasargv = hasattr(sys, 'argv')
+        # custom logon file (must be after creation of ROOT globals)
+        if hasargv and not '-n' in sys.argv and not self.PyConfig.DisableRootLogon:
+            rootlogon = os.path.expanduser('~/.rootlogon.py')
+            if os.path.exists(rootlogon):
+                # could also have used execfile, but import is likely to give fewer surprises
+                import imp
+                imp.load_module('rootlogon', open(rootlogon, 'r'), rootlogon, ('.py','r',1))
+                del imp
+            else:
+                # if the .py version of rootlogon exists, the .C is ignored (the user can
+                # load the .C from the .py, if so desired)
+                # system logon, user logon, and local logon (skip Rint.Logon)
+                name = '.rootlogon.C'
+                logons = [
+                    os.path.join(str(self.TROOT.GetEtcDir()), 'system' + name),
+                    os.path.expanduser(os.path.join('~', name))
+                    ]
+                if logons[-1] != os.path.join(os.getcwd(), name):
+                    logons.append(name)
+                for rootlogon in logons:
+                    if os.path.exists(rootlogon):
+                        self.TApplication.ExecuteFile(rootlogon)
+                del rootlogon, logons
```

Comment:
Also here, you don't have to `del` things, the GC should take care.

Reply:
Yes, will do that in a separate commit.